pkg/operator/sync.go

```diff
@@ -126,6 +144,70 @@ func (optr *Operator) syncTerminationHandler(config *OperatorConfig) error {
 	return nil
 }
 
+func (optr *Operator) syncWebhookConfiguration(config *OperatorConfig) error {
+	if err := optr.syncValidatingWebhook(
+		optr.kubeClient.AdmissionregistrationV1().ValidatingWebhookConfigurations(),
+		mapiv1.MachineValidatingWebhook(config.TargetNamespace),
+		mapiv1.MachineSetValidatingWebhook(config.TargetNamespace),
+	); err != nil {
+		return err
+	}
+
+	return optr.syncMutatingWebhook(
+		optr.kubeClient.AdmissionregistrationV1().MutatingWebhookConfigurations(),
+		mapiv1.MachineMutatingWebhook(config.TargetNamespace),
+		mapiv1.MacheSetMutatingWebhook(config.TargetNamespace),
+	)
+}
+
+func (optr *Operator) syncValidatingWebhook(client admissionclientv1.ValidatingWebhookConfigurationInterface, hooks ...admissionregistrationv1.ValidatingWebhook) error {
+	hook := mapiv1.NewValidatingWebhookConfiguration(hooks...)
```

Comment:
same than in https://github.com/openshift/machine-api-operator/pull/626/commits/264b18beb24d2fc30eb7d53d7e5f93572f64363c#r450938511